src/components/Home.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+  getToken,
+  words,
```

Comment:
'words' and 'calculateEstimate' appear to be unused variables, is this something that will be implemented later?

Reply:
Left them in because think they will be, especially 'calculateEstimate'. We can take them out later if we don't use 'em.

Reply:
I'm a fan of leaving out unused code/imports but this is benign, especially given we'll be using them soon.